language/Portuguese_Brazil/strings.po

```diff
@@ -1727,7 +1727,7 @@ msgstr "MAMEinfo"
 
 #: src/frontend/mame/ui/selmenu.cpp:132
 msgid "MARPScore"
-msgstr ""
+msgstr "MARPScore"
```

Comment:
Not necessarily - we don't necessarily translate proper nouns, and this is the name of a high score list.